content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -227,3 +243,184 @@ use our connector as an alternative payroll solution.
+each subcontractor is registered under the |CIS|. They are also responsible for maintaining records
+of all payments and deductions. Additionally, contractors must submit monthly returns to HMRC,
+including the following details:
+
+- Information about the subcontractors.
+- Records of payments made and any deductions applied.
+- A declaration confirming that the employment status of all subcontractors has been reviewed.
+- A declaration confirming that all subcontractors requiring verification have been verified.
+
+.. note::
+   If no payments were made to subcontractors in the previous tax month, contractors must notify
+   |HMRC| by the 19th of the month to avoid a penalty.
+
+.. _localization/united-kingdom/cis-configuration:
+
+Configuration
+-------------
+
+:ref:`Install <general/install>` the :guilabel:`UK - Construction Industry Scheme` module.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`UK - Construction Industry Scheme`
+     - `l10n_uk_reports_cis`
+     -  - Allows sending the Monthly return to |HMRC|
+        - CIS Deduction (GB) report for UK construction industry
+   * - :guilabel:`UK - HMRC API`
+     - `l10n_uk_hmrc`
+     -  - Includes the basics of |HMRC|.
+
+.. note::
+   Installing the :guilabel:`UK - Construction Industry Scheme` module installs the other module
+   simultaneously.
+
+To enable the :guilabel:`Test` mode and use test credentials, activate
+:ref:`developer mode <developer-mode>` and go to :menuselection:`Settings --> Technical -->
+System Parameters`. Then search for `l10n_uk_hmrc.api_mode`. Select it and switch the value from
+`production` to `test`.
+
+.. _localization/united-kingdom/cis-monthly-returns:
+
+Monthly returns
+---------------
+
+Monthly returns only work for vendor bills and vendor refunds. To submit a complete return to
+|HMRC|, several steps must be followed to report all payments made to subcontractors under the
+scheme during the previous tax month:
+
+- :ref:`localization/united-kingdom/cis-contractor-setup`
+- :ref:`localization/united-kingdom/cis-subcontractor-setup`
+- :ref:`localization/united-kingdom/cis-vendorbills`
+- :ref:`localization/united-kingdom/cis-monthly-return-sending`
+
+.. _localization/united-kingdom/cis-contractor-setup:
+
+Contractor setup
+~~~~~~~~~~~~~~~~
+
+To configure the company |HMRC| information, go to the :guilabel:`Settings` app. In the
+:guilabel:`Companies` section, click :guilabel:`Manage Companies` and select the company. Open the
+:guilabel:`HMRC` tab and configure the information in the :guilabel:`HMRC Credentials` and the
+:guilabel:`Contractor details` sections. All the fields are mandatory.
+
+.. _localization/united-kingdom/cis-subcontractor-setup:
+
+Subcontractor setup
+~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Contacts` app, open the subcontractor's contact form and select the
```

Comment:
I would remove "In the Contacts app" since we could access the contact's form from the Accounting app (e.g. though Accounting --> Vendors --> Vendors)
```suggestion
Access the subcontractor's contact form and select the
```